### PR TITLE
feat(Vue): add Select

### PR DESCRIPTION
**Pull request by wzpo**
image

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: vue/src/app/playground/index.vue [hunk 1]
```diff
@@ -1,11 +1,429 @@
 <template>
-  <div class="row">
-    <div class="example-spacing">
-      <!-- Insert Example Here -->
+  <div class="wrap">
```
Comment (mathsusan) [inline vue/src/app/playground/index.vue]: shouldn't this be left empty?  like we do in the other projects?  and these examples should be in example files

File: vue/src/lib/alert/examples/default.vue [hunk 2]
```diff
@@ -4,7 +4,7 @@
       <div class="row">
         <md-button
           ariaLabel="Click to Open"
-          @click="() => show = true"
+          @click="() => show = !show"
```
Comment (mathsusan) [inline vue/src/lib/alert/examples/default.vue]: this is for select or should some of these changes be in a separate PR?

File: vue/src/app/playground/index.vue (hunk 1 above)
Comment (wzpo) [reply vue/src/app/playground/index.vue]: Removed. This is the demo for engineers.

File: vue/src/lib/alert/examples/default.vue (hunk 2 above)
Comment (wzpo) [reply vue/src/lib/alert/examples/default.vue]: Reversed. Changed for demo last week.
